WcaOnRails/spec/factories/users.rb

```diff
@@ -170,6 +170,10 @@
       end
     end
 
+    trait :senior_delegate do
```

Comment:
This is not necessary, as you can simply use `FactoryBot.create :senior_delegate`.